src/facades/author.js

```diff
@@ -1,15 +1,13 @@
 'use strict'
 
-let books = require('../../db.json')
+let { getBooks } = require('./book')
 
-function getAuthors () {
-  let result = new Set(
-    books
-      .map(book => book.authors)
-      .reduce((prev, curr) => prev.concat(curr))
-      .sort()
-    )
-  return [...result]
+function getAuthors() {
```

Comment:
Yup; totally agree.
If our db was much larger this implementation would be horrible :)

I'll tackle later this week.
Cheers